app/assets/javascripts/miq_application.js

```diff
@@ -447,6 +447,10 @@ function miqButtonOnWhen(button, onwhen, count) {
 
 // Set the buttons in a div based on the count of checked items passed in
 function miqSetButtons(count, button_div) {
+  if (!miqDomElementExists('center_tb')) {
+    return
+  }
+
   if (button_div.match("_tb$")) {
```

Comment:
Should probably remove this `if` as well, since it should always be true..

And, maybe even remove the `button_div` arg? Either that or don't hardcode `center_tb` for `miqDomElementExists`..